### PR TITLE
improve documentation when vmware-iso is used on amazon-import post-p…

### PR DESCRIPTION
**Pull request by lmayorga1980**
This update includes some improved documentation on how to use vmware-iso and the amazon-import post-processor.

### CODE REVIEW

File: website/source/docs/post-processors/amazon-import.html.md [hunk 1]
```diff
@@ -104,6 +104,38 @@ Here is a basic example. This assumes that the builder has produced an OVA artif
 }
 ```
 
+## VMWare Example
+
+This is an example that uses `vmware-iso` builder and export the `.ova` file using ovftool.
```
Comment (SwampDragons) [inline website/source/docs/post-processors/amazon-import.html.md]: *exports

**Comment by rickard-von-essen** [comment]
> it looks like you have an extra set of brackets [] around the post-processors

@SwampDragons that is to make it a sequence, very important ([Docs: Post-Processors](https://www.packer.io/docs/templates/post-processors.html)).

File: website/source/docs/post-processors/amazon-import.html.md (hunk 1 above)
Comment (lmayorga1980) [reply website/source/docs/post-processors/amazon-import.html.md]: fixed

**Comment by lmayorga1980** [comment]
@SwampDragons based on the following thread i was able to make it work and that is the reasy why i am using double brackets.

https://github.com/hashicorp/packer/issues/4832#issuecomment-297167073

**Comment by SwampDragons** [comment]
Fair enough! This is what I get for only skimming the code and not trying it out :)

**Pull request merged**